README.md

```diff
@@ -67,6 +72,10 @@ Having a bug tracking system doesn't make bug handling efficient and CI/CD doesn
 - What's the maturity stage? (early finding direction / feature work / maintenance / ...)
 - Can I contribute to FOSS projects? Are there any approvals needed?
 - Are there any non-compete or non-disclosure agreements I'll be asked to sign?
+- What is your approach to working overtime? How often it is required to work overtime and what are the compensation rules for it?
+- Is there possibility to change projects / roles?
+- Do you organise internal/public meetups / attend conferences?
+- Is it allowed to allocate N hours per week for career development purposes?
```

Comment:
I would say this one is a bit too vague. What do you mean?

Reply:
It is meant to be open-ended question. I'll give you example from my current company: during each week, we can allocate 2 hours for non-sprint related stuff - reading blogs, contributing to open source, doing some courses etc...

Reply:
And may I ask if you're able to do those things? I know some times is hard to make a case for things which don't have a direct impact on the business